nemo/collections/nlp/models/duplex_text_normalization/duplex_decoder.py

```diff
@@ -50,6 +51,22 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         # Language
         self.lang = cfg.get('lang', None)
 
+        # Covering Grammars
+        self.cg_normalizer = None  # Default
+        # We only support integrating with English TN covering grammars at the moment
+        self.use_cg = cfg.get('use_cg', False) and self.lang == constants.ENGLISH
+        if self.use_cg:
+            self.setup_cg(cfg.get('neural_confidence_threshold', 0.99))
```

Comment:
setup_cgs

Reply:
why setting up the grammars twice? first here and also in helpers?

Reply:
So models that were trained before this change do not have this field (and so its default value will always be `false` no matter what is set later in the config file). This is a hacky way to allow previously trained models to be able to use CGs. 

I will think about a cleaner way.

Reply:
ok, thanks for explaining. That is ok then. We will delete later